client.go

```diff
@@ -143,6 +143,92 @@ func (c *client) createSP(
 	return &result, password, err
 }
 
+// addAppPassword adds a new password to an App's credentials list.
+func (c *client) addAppPassword(ctx context.Context, appObjID string, duration time.Duration) (string, string, error) {
+	keyID, err := uuid.GenerateUUID()
+	if err != nil {
+		return "", "", err
+	}
+
+	// Key IDs are not secret, and they're a convenient way for an operator to identify Vault-generated
+	// passwords. These must be UUIDs, so the three leading bytes will be used as an indicator.
+	keyID = "ffffff" + keyID[6:]
+
+	password, err := uuid.GenerateUUID()
+	if err != nil {
+		return "", "", err
+	}
+
+	now := time.Now()
```

Comment:
Pretty sure this will grab the current time in whatever timezone the server is in that's hosting the application. If the server were in US Eastern time, would everything still work as expected? Especially if the remote Azure server were set to a different timezone?

Reply:
It will probably work OK since what gets sent is a full RFC3339 timestamp, and one of their samples shows time.Now() being used. But the `az` CLI tool sends UTC, and passwords created in the portal get UTC, so UTC it is. 🙂

Reply:
Cool cool, I figured it did something like that but just wanted to ask. :-)